pull_request_template.md

```diff
@@ -0,0 +1,58 @@
+## High Level Description
+
+<!--
+Please explain what this bounty is for in 1-3 sentences.
+
+Ex. "Build an integration with Shopify that allows businesses to easily add XRP as a payment option next to credit card payment flows."
+-->
+
+Description:
+
+<!--
+Please indicate what you think a fair payment for completion of this bounty would be. The exact amount will be determined by the approving committee and may be higher or lower than the initial estimate
+-->
+
+Amount: $
+
+### Audience
+
+<!--
+Please check relevant options and delete irrelevant ones.
+
+- [X] = Checked
+- [ ] = Unchecked
+-->
+
+- [ ] First time XRPL builders
+- [ ] Experienced XRPL builders
+- [ ] CUSTOM - Replace this with your own category
```

Comment:
I don't think these need to be checkboxes. Bounties can just have a few bullet points indicating the relevant audience